src/open_dread_rando/misc_patches/tilegroup_patcher.py

```diff
@@ -21,3 +22,22 @@ def patch_tilegroup(editor: PatcherEditor, group: dict):
     gridTiles = actor.pComponents.TILEGROUP.aGridTiles
     for tile in gridTiles:
         tile.eTileType = group["tiletype"]
+
+def patch_individual_tiles(editor: PatcherEditor, actor_reference: dict, tiles: list):
+    actor = editor.resolve_actor_reference(actor_reference)
+
+    if not is_tilegroup(actor):
+        raise ValueError(f"Actor at {actor_reference} is not a breakable tile group.")
+
+    gridTiles = actor.pComponents.TILEGROUP.aGridTiles
+
+    if len(tiles) != len(gridTiles):
+        tiles_string = f"got {len(tiles)}, expected {len(gridTiles)}"
+
+        if len(tiles) < len(gridTiles):
+            raise ValueError(f"Too few tiles provided to patch tilegroup {actor_reference} ({tiles_string})")
+        else:
+            LOG.warning(f"More tiles provided than in tilegroup {actor_reference} ({tiles_string})")
```

Comment:
I think you got it backwards - having too many tiles should raise an error for sure

Reply:
No, because it iterates over the actor's tiles, not the input's tiles. If too many are provided they're unused, while too few is an error.

Reply:
oh, I see. the original condition errored if there were too many tiles and did nothing with too few so I didn't realize that it also had it backwards